src/backend/distributed/utils/citus_outfuncs.c

```diff
@@ -418,6 +445,34 @@ OutTask(OUTFUNC_ARGS)
 
 
 void
+OutTaskExecution(OUTFUNC_ARGS)
+{
+	int count = 0;
+	WRITE_LOCALS(TaskExecution);
+	WRITE_NODE_TYPE("TASKEXECUTION");
+
+	count = node->nodeCount;
```

Comment:
`count` is a bit generic. Maybe we can just use `node->nodeCount` directly in `WRITE_ENUM_ARRAY`?